roxctl/common/client.go

```diff
@@ -103,6 +109,10 @@ func (client *roxctlClientImpl) DoReqAndVerifyStatusCode(path string, method str
 
 // Do executes a http.Request
 func (client *roxctlClientImpl) Do(req *http.Request) (*http.Response, error) {
+	if RoxctlCommand != "" {
+		req.Header.Add(roxctlCommandHeader, RoxctlCommand)
```

Comment:
Do we need roxctl version?

Reply:
No, the version is already provided in the `User-Agent` header.